### ISSUE TITLE
Re-submit queue item after fixing issues with address 

### ISSUE THREAD

**Issue by oleksandr-butenko** [classyllama/ClassyLlama_AvaTax#235]
Customer placed an order. 
And for what ever reason - zip code is not valid. 
Client is not using validation of address on checkout. 
Client would prefer to fix issues after placing an order. 

### Preconditions
1. classyllama/module-avatax -  version:1.4.9
2. Magento 2.2.7 
3. PHP 7.1.X

### Steps to reproduce
1. Address validation by avalara API is disabled on checkout. 
2. Customer placed an order with incorrect zip code. 
3. During processing queue invoice for this order is getting "An error occurred when attempting to send Invoice #1XXXXXXX to AvaTax. Error: Result code: Error Message: Name: JurisdictionNotFoundError Summary: Unable to determine the taxing jurisdictions. Details:... "
4. Admin user is getting notification like "The AvaTax Queue has 1 document(s) that failed during processing and were not successfully sent to AvaTax. Check the queue"
5. Admin user is going to order and fixing this mistake in the order address. 

### Expected result
1. Queue item moved to status  status failed with maximum number of processing attempts. 

### Actual result 
1. After fixing address issues we should have an ability to resubmit queue item - so it will be processed. 

Basically now I'm going to DB and changing status of that item to "pending" and setting number of attempts to 0. 
`UPDATE avatax_queue t SET t.queue_status = 'pending', t.attempts = 0, t.message = null WHERE t.queue_id = XXXXX`

**Comment by erikhansen** [comment]
Avalara has shifted how they are handling support for this extension and are no longer providing support via Github issues. If you are still experiencing this issue, please contact support@avalara.com.

Sorry for the delay in replying to this issue.
